config/schema/field_definitions.json

```diff
@@ -98,6 +98,10 @@
     }
   },
 
+  "detailed_format": {
+    "type": "identifier"
```

Comment:
Could you add a "description" entry in this field definition, please?  They're optional, and for documentation only, but it's very helpful to have one indicating what sort of data is expected in the field, and where it comes from.  I'm trying to make sure any new fields have one, and then add it for existing fields.